src/test/renderer/self/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd ${0%/*}
+node ../../../../bin/typedoc --name "TypeDoc Documentation" --tsconfig ../../../../tsconfig.json --readme ../../../../README.md --out doc/ --entryPointStrategy expand ../../../../src/lib/
```

Comment:
I think we can just delete this folder now, there's a typedoc.json under .config which is used to create the docs on the website